src/HomeAssistantManager.cpp

```
@@ -849,6 +851,36 @@ void HomeAssistantManager::handleCommand(const char* topic, const char* payload)
         return;
     }
 
+    // deduct: store filament usage deduction — tag may not be on scanner
+    if (strcmp(command, "deduct") == 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)